docs/src/main/asciidoc/spring-web-guide.adoc

```diff
@@ -153,14 +153,14 @@ Open your browser to http://localhost:8080/greeting/world.
 
 The result should be: `{"message": "HELLO WORLD!"}`.
 
-== Run the application as a native
+== Run the application as a native executable
 
-You can of course create a native image using instructions similar to link:building-native-image-guide.html[this] guide.
+You can of course create a native image using the instructions of the link:building-native-image-guide.html[Building a native executable guide].
 
-== Supported Spring Web functionality
+== Supported Spring Web functionalities
```

Comment:
I'm not sure about this one, I personally wouldn't use the plural form, but if you think it's better, fine by me

Reply:
I checked here before adding the comment: https://www.merriam-webster.com/dictionary/functionality and they have an example saying "data management functionalities such as data integrity, security, recoverability and manageability".

I also found "The software update adds interesting functionalities." here https://www.linguee.fr/anglais-francais/traduction/functionality.html .

We are talking about the supported features so plural it should be IMHO.

Reply:
Cool!